testing/environments/alpha2.yml

```diff
@@ -4,7 +4,7 @@
 elasticsearch:
   build: ./docker/elasticsearch
   dockerfile: Dockerfile-5.0.0-alpha1    # still using alpha1 because alpha2 doesn't work fine in docker
-  command: elasticsearch -Ees.network.host=0.0.0.0 -Ees.discovery.zen.minimum_master_nodes=1
+  command: elasticsearch -Enetwork.host=0.0.0.0 -Ediscovery.zen.minimum_master_nodes=1
```

Comment:
I'm not sure, but I think alpha2 doesn't work because of the system checks. Did you try it? If it doesn't work, we could just remove the files.